### PR TITLE
Hellfire missiles

### PR DESCRIPTION
**Pull request by PabstMirror**
Adds laser guided hellfire (K version) - weapon, magazines, ammo
User selectable attack profiles (LOBL/LOAL-DIR, LOAL-HI, LOAL-LO) via interaction menu
Simple hud addition to show attack mode, laser code and laser seeker status before firing

image

The weapon and magazine should be usable on any vehicle (hud and actions are created dynamically).
For now I've put them onto a 2nd version of the Comanche.

### CODE REVIEW

**Comment by gienkov** [comment]
How about implementing only realistic codes like those mentioned here: https://github.com/acemod/ACE3/issues/967

**Comment by PabstMirror** [comment]
From what I've read, the K varient can lock onto PRF codes, 
so I'm not sure if Ocelotr's comment is correct.

**Comment by gienkov** [comment]
It's just cosmetic change so it's not important anyway, it's good to have that functionality!

**Comment by gienkov** [comment]
Here are some interesting info:
http://www.globalsecurity.org/military/library/policy/usmc/mcwp/3-16/fdraft_appk.pdf

> Hellfire can be launched in four firing modes: one
missile (single), two or more missiles on the same
code (rapid), two or more missiles launched on
different codes using multiple laser designators
(ripple), or multiple codes and designators used in
combination of rapid and ripple fire. The PRF code is
cockpit selectable.

File: docs/wiki/feature/hellfire.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+version:
+  major: 3
+  minor: 9
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Guidance
+Hellfire missile is Semi-active laser guided; requiring an observer to guide the missile for it's entire flight.
```
Comment (jonpas) [inline docs/wiki/feature/hellfire.md]: Lower-case `semi`?

File: docs/wiki/feature/hellfire.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- LOBL: Lock-On-Before-Launch, standard top attack.
+- LOAL-DIR: Missile flies with a low altitude until acquiring a laser.
+- LOAL-LOW: Missile immediately gains ~90m altitude.
+- LOAL-HI: Missile immediately gains ~300m altitude.
+
+## 2. Usage
+- Switching to the hellfire weapon will show additional information about the weapon in weapon status display.
+- Shows: lock mode, laser code and a laser receiver indicator.  E.G. `LOAL-DIR    CODE: 1111`
+- Laser receiver indicator turns red when it detects a laser pulse set the the current code.
+- Cycle attack profiles with vehicle's ace_interaction menu or with the missile guidance key (default: <kbd>Ctrl</kbd> + <kbd>Tab</kbd>)
```
Comment (jonpas) [inline docs/wiki/feature/hellfire.md]: `vehicle's ACE3 Interaction Menu`

File: docs/wiki/feature/hellfire.md (hunk 1 above)
Comment (Drofseh) [inline docs/wiki/feature/hellfire.md]: its entire flight

**Comment by Glowbal** [comment]
@jonpas could you take another look at this?

**Pull request merged**